docs/api.rst

```diff
@@ -0,0 +1,39 @@
+API
+===
+.. contents::
+    :local:
+    :backlinks: none
+
+Note: All interfaces not documented here are considered to be private.
+
+Client
+------
+.. automodule:: amazon_transcribe.client
+    :members:
+    :undoc-members:
+
+
+Response Handlers
+-----------------
+.. automodule:: amazon_transcribe.handlers
+    :members:
+    :undoc-members:
+
+
+Transcription Model
+-------------------
+.. autoclass:: amazon_transcribe.model.AudioStream
```

Comment:
We should document all of the modeled shapes as customers will need to interact with them.

Reply:
Everything should be typed out and updated now.